### ISSUE TITLE
Needs to be role aware

### ISSUE THREAD

**Issue by btbonval** [medic/medic-webapp#2234]
If the `full_access` role exists, then all `CREATE` statements need to be prefixed with `SET ROLE full_access`.

Otherwise the tables will be owned by whatever user the adapter is logging in as. `full_access` is not a role which will have login access (and thus acts more like a group than a user), but using `SET ROLE full_access` allows the tables to be modified by anyone in the `full_access` group.

**Comment by btbonval** [comment]
The "if" above is a lot of extra logic and test code for no good reason. I'm quite content to require that a `full_access` role be present, and I will add documentation asserting as much.

**Comment by btbonval** [comment]
Commit came back green. This issue should be covered now.

I updated package.json to 0.0.3 since this might break previous installations that don't use `full_access` role properly. A 0.0.2 tag was added beforehand so those systems have an easy checkout option.

**Comment by btbonval** [comment]
@garethbowen sorry I got used to my way of doing things where nobody knows what I'm doing with the adapter. I might have jumped the gun on closing the issue. I just realized this would be a good opportunity for code review, as I just added a new feature which includes unit and integration tests.

I'm assigning to you for code review but feel free to pass the buck to whomever will be doing adapter stuff. Commit link is referenced above.
